### DISCUSSION TITLE
Trying to create a mastodon.online account

### DISCUSSION THREAD

**Discussion by kbeams1** [mastodon/mastodon#23179] [Q&A] [answered]
When going to make an account it provides me with :
" Creating an account on mastodon.online is currently not possible. "
Anyway to fix this ?

**Comment by StrangeMinds1960** [comment]
This has nothing to do with the Mastodon software itself.  You need to ask those that run mastodon.online, they're the ones that have disabled signups for whatever reason

**Reply by kbeams1** [reply]
How would I get in contact with them ?

**Reply by StrangeMinds1960** [reply]
https://mastodon.online/about

**Reply by melroyvandenberg** [reply]
Don't contact him (he is the creator of Mastodon). The server is overloaded with new users. He was I think forced to close down registrations. 

Just try to register an user on another server. Or ideally host your own instance!

**Comment by ClearlyClaire** [comment, marked as answer]
This means registrations are currently disabled on `mastodon.online`. Please try to find another server which has open registrations.

**Comment by melroyvandenberg** [comment]
@kbeams1 Can you close the discussion again?
